app/scenes/AboutApp/index.js

```diff
@@ -19,7 +19,7 @@ class AboutApp extends Component { // eslint-disable-line
     render() {
         const { skin } = this.props;
         const title = skin ? skin.name : 'MindLogger';
-        if (typeof skin.about !== 'undefined') {
+        if (typeof skin.about !== 'undefined' && skin.about.replace(/\s/g, '').length) {
```

Comment:
I would probably break this into two lines for clarity. I had to google what `/\s/g` was...

Reply:
Okay, this should be better now. I also added prop types and fixed some formatting.